includes/wikia/api/SendGridPostBackApiController.class.php

```diff
@@ -1,91 +1,115 @@
 <?php
 /**
- * API which SendGrid will post data to when an email is opened, clicked, bounced back, unsubscribed, marked as spam, etc..
+ * API which SendGrid will post data to when an email is opened, clicked, etc..
  *
- * @author Piotr Molski ( moli@wikia-inc.com )
+ * For more information see:
+ *
+ *   https://sendgrid.com/docs/API_Reference/Webhooks/event.html
  */
 
-class SendGridPostbackController extends WikiaApiController {
+use Wikia\Logger\WikiaLogger;
```

Comment:
So this isn't super important, but another approach is to use the `use Loggable` trait inside the class. When you do that, you have access to logging methods like:
```php
$this->debug();
$this->info();
$this->warn();
```
It's pretty nifty.